src/olympia/templates/header_title.html

```diff
@@ -1,8 +1,10 @@
 <h1 class="site-title">
-{% macro heading(text, icon) -%}
+{% macro heading(app, text, icon) -%}
```

Comment:
What purpose does this `app` serve? I don't see it referenced anywhere.

Reply:
I copied that over from the duplicate file, but it seems useless yeah. I'll try removing it